tests/integration/operator/ginkgo/installation_test.go

```diff
@@ -405,7 +405,7 @@ type PorterCfgOpts struct {
 	SecretsNamespace string
 }
 
-//NewPorterCofnig GetDefaultPorterConfig - DefaultStorage, DefaultSecretsPlugin and DefaultStorageConfig set.  We tweak DefaultSecrets and SecretsConfig as needed
+// NewPorterCofnig GetDefaultPorterConfig - DefaultStorage, DefaultSecretsPlugin and DefaultStorageConfig set.  We tweak DefaultSecrets and SecretsConfig as needed
```

Comment:
If we're in here touching this might as well fix the typo :)

Reply:
Hah, oops I am just awful at spotting typos. 👍